hack/Dockerfile.golang

```diff
@@ -0,0 +1,50 @@
+# syntax=docker/dockerfile:1.5-labs
+
+ARG BASE_IMAGE=fedora:36
+FROM --platform=$TARGETPLATFORM ${BASE_IMAGE} as base
+
+ARG GO_VERSION=1.20.5
```

Comment:
Any reason to bump go version to 1.20? I am asking because the project is still configured to 1.19.

Reply:
There is a difference between the version of the used Go toolchain and the version declared in the go.mod.

The version in the go.mod is a _minimal_ version and specifies which language features are assumed to exist.

Afaik there is no reason ever not to use the latest toolchain, especially as this ensures that security fixes to the standard library are included in the code.

Reply:
Sorry, I don't know much about backward compatible in "Gopshere". If there is any problem then picking the latest and greatest stable version (1.120.5) is fine.